packages/rocketchat-lib/server/startup/oAuthServicesUpdate.js

```diff
@@ -0,0 +1,103 @@
+/* globals CustomOAuth */
+const logger = new Logger('rocketchat:lib', {
+	methods: {
+		oauth_updated: {
+			type: 'info'
+		}
+	}
+});
+
+function debounce(fn, delay) {
```

Comment:
Can you try with `_.debounce`? I think it will not be a problem, we use it in some places.